WcaOnRails/config/application.rb

```diff
@@ -14,6 +14,17 @@
 ENVied.require(*ENV['ENVIED_GROUPS'] || Rails.groups)
 
 module WcaOnRails
+  AVAILABLE_LOCALES = {
```

Comment:
Just my personal preference, but I like keeping it in the code. It's hardcoded wherever it is. The simplest way to get data into Ruby is to write it in Ruby.

Maybe it should be in its own `locales.rb` file.

I don't know how the load order affects things, but it's described here: http://guides.rubyonrails.org/initialization.html